src/engine/ruleset/integrations/apache-http/changelog.yml

```diff
@@ -0,0 +1,4 @@
+items:
+  - version: 1.0.0-dev
+    short: Created integration for apache http server
+    pr: "[#PR0000](#)"
```

Comment:
Add the PR.

Reply:
[#16910](https://github.com/wazuh/wazuh/pull/16910)

both changelog and readme updated